lib/internal/bootstrap_node.js

```diff
@@ -361,20 +361,19 @@
         }
 
       } else {
-        // If we handled an error, then make sure any ticks get processed
-        NativeModule.require('timers').setImmediate(process._tickCallback);
-
         // Emit the after() hooks now that the exception has been handled.
         if (async_hook_fields[kAfter] > 0) {
           do {
             NativeModule.require('async_hooks').emitAfter(
               async_uid_fields[kCurrentAsyncId]);
-          // popAsyncIds() returns true if there are more ids on the stack.
-          } while (popAsyncIds(async_uid_fields[kCurrentAsyncId]));
+          } while (asyncIdStackSize() > 0);
```

Comment:
> Ack, but if you do it just before the loop it (1) looks cleaner [to me] (2) still lazy enough.

do you mean creating `let timers = null;` in `setupProcessFatal()` then doing something like?:
```js
if (timers === null)
  timers = NativeModule.require('timers');
```
Or just `const timers = NativeModule.require('timers');` before the loop every time?